### PR TITLE
Astro 5/26 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -16,6 +16,20 @@ If you have any questions or a bug to report, don't hesitate to reach out to [As
 
 **Latest CLI Version**: 1.5.0 ([Release notes](cli-release-notes.md))
 
+## May 26, 2022
+
+### New Datasets Page for Data Lineage
+
+You can now use the new **Datasets** page in the **Lineage** tab to view a table of the most recent datasets that have been either accessed or updated by your data pipelines. This page can help you get a high level overview over which datasets your pipelines are currently accessing.
```
Comment (paolaperaza) [inline astro/release-notes.md:23]:
```suggestion
You can now use the new **Datasets** page in the **Lineage** tab of the Cloud UI to view a table of the most recent datasets that have been either accessed or updated by your data pipelines. This page can help you get a high-level overview of which datasets your pipelines are currently accessing.
```
```suggestion
You can now use the new **Datasets** page in the **Lineage** tab to view a table of the most recent datasets that have been either accessed or updated by your data pipelines. This page can help you get a high level overview over which datasets your pipelines are currently accessing.
```

File: astro/release-notes.md [hunk 2]
```diff
@@ -16,6 +16,20 @@ If you have any questions or a bug to report, don't hesitate to reach out to [As
...
+
+![Datasets page](/img/release-notes/datasets-page.png)
+
+To learn more about a recent dataset change, you can click the dataset's name in the table. This shows the dataset within the context of its lineage graph.
```
Comment (paolaperaza) [inline astro/release-notes.md:27]:
```suggestion
To learn more about a recent dataset change, you can click the dataset's name in the table. This shows the dataset within the context of its lineage graph. For more information, see [Data Lineage on Astro](https://docs.astronomer.io/astro/data-lineage).
```

Comment (paolaperaza) [inline astro/release-notes.md:27]:
```suggestion
To learn more about a recent dataset change, you can click the dataset's name in the table. This shows the dataset within the context of its lineage graph. For more information on the lineage tab on Astro, see [Data Lineage on Astro](https://docs.astronomer.io/astro/data-lineage).
```
```suggestion
To learn more about a recent dataset change, you can click the dataset's name in the table. This shows the dataset within the context of its lineage graph.  
```

File: astro/release-notes.md [hunk 3]
```diff
@@ -16,6 +16,20 @@ If you have any questions or a bug to report, don't hesitate to reach out to [As
...
+
+You can now use the new **Datasets** page in the **Lineage** tab to view a table of the most recent datasets that have been either accessed or updated by your data pipelines. This page can help you get a high level overview over which datasets your pipelines are currently accessing.
+
+![Datasets page](/img/release-notes/datasets-page.png)
+
+To learn more about a recent dataset change, you can click the dataset's name in the table. This shows the dataset within the context of its lineage graph.  
+
+### Bug Fixes
+
+- Fixed an issue where Deployments using an unsupported version of Astro Runtime appeared with a Runtime version of **Unknown**
```
Comment (paolaperaza) [inline astro/release-notes.md:31]:
```suggestion
- Fixed an issue where the **Astro Runtime** field in the Deployment view of the Cloud UI had a value of **Unknown** for Deployments using an unsupported version of Astro Runtime
```

Comment (paolaperaza) [inline astro/release-notes.md:31]:
```suggestion
- Fixed an issue where the **Astro Runtime** field of the Cloud UI listed the running version as **Unknown**  for Deployments using an unsupported version of Astro Runtime
```
```suggestion
- Fixed an issue where Deployments using an unsupported version of Astro Runtime appeared with a Runtime version of **Unknown**
```

**Review by paolaperaza — APPROVED** [review]

File: astro/release-notes.md (hunk 2 above)
Comment (conorbev) [reply astro/release-notes.md:27]: I suggest keeping it simpler:

`Click on the name of a dataset to show its lineage graph. For more information, see [Data Lineage on Astro](data-lineage.md).`

I think the fact that we show an "updated at" column may actually be confusing matters a bit. It's not necessarily that the dataset was modified at that time but just that an airflow task read to it or wrote to it so I wouldn't focus so much on the "change" ( though certainly if there are actual changes the compare view should surface them) and instead just think of the datasets page as a way of finding data that your tasks interact with and viewing the lineage of those interactions.

Comment (paolaperaza) [reply astro/release-notes.md:27]: Nice makes sense - adjusted

File: astro/release-notes.md (hunk 1 above)
Comment (kwcanuck) [inline astro/release-notes.md:23]:
```suggestion
The new **Datasets** page in the **Lineage** tab of the Cloud UI lists the most recent datasets that your DAGs have read or written to. This information can help you quickly identify dataset dependencies and data pipeline access requirements.
```

**Pull request merged**